vite.config.js

```diff
@@ -1,32 +1,61 @@
+import dotenv from 'dotenv'
 import { defineConfig } from 'vite'
+import { createHtmlPlugin } from 'vite-plugin-html'
 import path from 'path'
 import pluginRewriteAll from 'vite-plugin-rewrite-all'
 import svgLoader from 'vite-svg-loader'
 import vue from '@vitejs/plugin-vue'
 
+/** @typedef {import('./src/types/index').PathConfig} PathConfig */
+
+dotenv.config()
+
+/**
+ * @type {PathConfig}
+ */
+const PATH_CONFIG_DEFAULT = {
+  baseGuiPath: '/gui',
+  // **TIP**: Change this value to test various GUI base path scenarios.
+  // baseGuiPath: '/dev/gui',
+  apiUrl: /** @type {string} */ (process.env.VITE_KUMA_API_SERVER_URL),
```

Comment:
Just incase, I think the default for this in prod from the backend is `/` (the env var looks to be full hostname:port) i.e. without the hostname:port. Should we mirror that so its just `/` here also? Not sure if there is a restriction on running msw on the same port as vite, I would guess not?

Reply:
Let me try this in Kuma with a local build. I think this should work.

In our `RestClient` class, we’d always set the correct URL during development (where the API origin is _different_ from the GUI origin) and in production we’d use `window.location.origin`. I’ve made those changes in https://github.com/kumahq/kuma-gui/pull/474/commits/66ca34e7dfc81d0351be44d73f7edbf7cf579afe.

Reply:
Perhaps we can get rid of this environment variable by proxying requests from 8080 to 5681 during development, but I’ll consider this separately from this PR.